MapboxCoreNavigation/MGLMapboxEvents.swift

```diff
@@ -0,0 +1,30 @@
+import Mapbox
+import Polyline
+
+extension MGLMapboxEvents {
+    public class func addDefaultEvents(routeProgress: RouteProgress, sessionIdentifier: UUID, sessionNumberOfReroutes: Int = 0) -> [String: Any] {
```

Comment:
Not really, because MapboxCoreNavigation and MapboxNavigation are separate modules. If you were to declare this method in an Objective-C header but implement it in Swift, it would be possible to give the header “private” membership in the MapboxCoreNavigation target, which would make it accessible to other targets in the same Xcode project, like MapboxNavigation. At least that’s true in Xcode and Carthage land. I don’t think that would work with CocoaPods.